src/config.ts

```diff
@@ -1,3 +1,25 @@
 export const TCR_CONTRACT_ADDRESS = process.env.TCR_CONTRACT_ADDRESS
 export const TCR_LIST_ID = Number(process.env.TCR_LIST_ID) || 0
 export const TCR_CACHE_TIME = 5 * 60 * 1000
+export const TOKEN_OVERRIDES = {
+  1: {
+    // Deprecated sUSD proxy contract
+    '0x57ab1e02fee23774580c119740129eac7081e9d3': {
+      forceAddressDisplay: true,
```

Comment:
I personally like if we call it with as a property name, instead of an "action".  In this case it's a flag for `deprecated` token.

Reply:
Hmmm what would you rather call this?

Reply:
mm, that would be expeculatory future need, not sure it would make sense to show a known address out of the TCR. It would mean that we add a list on top of the TCR, and I think that's something we try to avoid.

Modeling properties let us describe the data, so later we can represent it. Model it as "actions" like force display, makes the data to be coupled to its use. Make it weird if suddenly there's more uses for the same concept

Reply:
Anyways, unimportant! so good for me 😃